tests/svg_test.py

```diff
@@ -487,3 +487,23 @@ def test_apply_gradient_translation(gradient_string, expected_result):
     etree.cleanup_namespaces(svg.svg_root)
 
     assert etree.tostring(el).decode("utf-8") == expected_result
+
+
+@pytest.mark.parametrize(
+    "svg_content, expected_result",
+    [
+        # Blank fill
+        # https://github.com/googlefonts/nanoemoji/issues/229
+        (
+            '<path fill="" d=""/>',
+            (SVGPath(),),
+        ),
+    ],
+)
+def test_default_for_blank(svg_content, expected_result):
+    svg_string = (
```

Comment:
the svg_test_helpers.py module contains an `svg_string` method that does exactly that (i.e. create an svg root element and interpolate the svg content in it), it's already star-imported at the top, you may want to use it here

Reply:
Good catch, ty